flink-python/dev/lint-python.sh

```diff
@@ -0,0 +1,508 @@
+            if [ $? -ne 0 ]; then
+                echo "Failed to install brew"
+                exit 1
+            fi
+        fi
+
+        hash "wget" 2>/dev/null
+        if [ $? -ne 0 ]; then
+            brew install wget 2>&1 >/dev/null
+            if [ $? -ne 0 ]; then
+                echo "Failed to install wget"
+                exit 1
+            fi
+        fi
+    fi
+}
+
+# The script choose miniconda as our package management tool.
+# The script use miniconda to create all kinds of python versions and
+# some pakcages such as tox and flake8.
+
+function install_miniconda() {
+    OS_TO_CONDA_URL=("https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh" \
+        "https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh")
+    print_function "STEP" "download miniconda..."
+    if [ ! -f "$CONDA_INSTALL" ]; then
+        download ${OS_TO_CONDA_URL[$1]} $CONDA_INSTALL_SH
+        chmod +x $CONDA_INSTALL_SH
+        if [ $? -ne 0 ]; then
+            echo "Please mannually chmod +x $CONDA_INSTALL_SH"
+            exit 1
+        fi
+        if [ -d "$CURRENT_DIR/.conda" ]; then
+            rm -rf "$CURRENT_DIR/.conda"
+            if [ $? -ne 0 ]; then
+                echo "Please mannually rm -rf $CURRENT_DIR/.conda directory.\
+                Then retry to exec the script."
+                exit 1
+            fi
+        fi
+    fi
+    print_function "STEP" "download miniconda... [SUCCESS]"
+
+    print_function "STEP" "installing conda..."
+    if [ ! -d "$CURRENT_DIR/.conda" ]; then
+        $CONDA_INSTALL_SH -b -p $CURRENT_DIR/.conda 2>&1 >/dev/null
+        if [ $? -ne 0 ]; then
+            echo "install miniconda failed"
+            exit $CONDA_INSTALL_STATUS
+        fi
+    fi
+    print_function "STEP" "install conda ... [SUCCESS]"
+}
+
+# Install some kinds of py env.
+function install_py_env() {
+    py_env=("2.7" "3.3" "3.4" "3.5" "3.6" "3.7")
+    for ((i=0;i<${#py_env[@]};i++)) do
+        if [ -d "$CURRENT_DIR/.conda/envs/${py_env[i]}" ]; then
+            rm -rf "$CURRENT_DIR/.conda/envs/${py_env[i]}"
+            if [ $? -ne 0 ]; then
+                echo "rm -rf $CURRENT_DIR/.conda/envs/${py_env[i]} failed, please \
+                rm -rf $CURRENT_DIR/.conda/envs/${py_env[i]} mannually.\
+                Then retry to exec the script."
+                exit 1
+            fi
+        fi
+        print_function "STEP" "installing python${py_env[i]}..."
+        ${CONDA_PATH} create --name ${py_env[i]} -y -q python=${py_env[i]} 2>&1 >/dev/null
+        if [ $? -ne 0 ]; then
+            echo "conda install ${py_env[i]} failed.\
+            You can retry to exec the script."
+            exit 1
+        fi
+        print_function "STEP" "install python${py_env[i]}... [SUCCESS]"
+    done
+}
+
+# Install tox.
+# In some situations,you need to run the script with "sudo". e.g. sudo ./lint-python
```

Comment:
execute "sudo ./lint-python" will print this message on my device: "sudo: ./lint-python: command not found".
Is it mean "sudo ./lint-python.sh" ?

Reply:
Yes, `sudo ./lint-python.sh`.